src/main.cpp

```diff
@@ -450,6 +573,40 @@ class gfxPipelineHandler {
         UpdateWindow(hWND);
         SetFocus(hWND);
     }
+
+    static void Install() {
+        InstallPatch("Enables pointer in windowed mode.", { 0x90, 0x90 }, {
+            0x4F136E,
+        });
+
+        if (!datArgParser::Get("16bit")) {
```

Comment:
When SetRes gets rewritten, I say we combine cdepth and zdepth into one parameter. Why would anyone have a 16-bit color depth, but a 32-bit z-depth? Makes no sense to me.